calico/datamodel_v1.py

```diff
@@ -85,6 +86,10 @@
 ENDPOINT_STATUS_DOWN = "down"
 ENDPOINT_STATUS_ERROR = "error"
 
+# Information intended for use by the DHCP agent.
+DHCP_DIR = ROOT_DIR + "/dhcp" + DHCP_VERSION
+SUBNET_DIR= DHCP_DIR + "/subnet"
```

Comment:
But would such config go under the 'dhcp' subtree, with 'dhcp' being explicitly specified as information for the DHCP agent?